src/config/generator.js

```diff
@@ -68,17 +71,20 @@ class ConfigGenerator {
   }
 
   static showGenerationSummary () {
-    console.log('=================================================')
-    console.log('         Config generated successfully!          ')
-    console.log('=================================================')
-    console.log(`Root Directory : ${process.cwd()}`)
-    console.log('   |- .appstrap')
-    console.log('      |- presets/')
-    console.log('      |- config.js')
-    console.log('')
-    console.log('For more information visit the following links:')
-    console.log('Getting started: https://some-getting-started-link-here')
-    console.log('Documentation: https://appstrap-documentation-link-here')
+    console.log(chalk`
+    ===============================================================
+      {yellow.bold Appstrap} Config generated {green.bold successfully!} 
+    
+      Root Directory : ${process.cwd()} {cyan
+         |- .appstrap
+            |- presets/
+            |- config.js}
+
+      For more information visit the following links:
+      Getting started: {blue https://some-getting-started-link-here}
```

Comment:
We do have a README on the repository. As well as an API, maybe referencing that would be a good option for now?

Reply:
Getting Started: https://github.com/pixelprodotco/appstrap#getting-started
Documentation: https://github.com/pixelprodotco/appstrap/blob/master/docs/api.md#appstrap-api

Reply:
Good catch! If @pixelprodotco agrees I can add those in this PR as well. Otherwise we can also keep these changes separate and add those links later